client/webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const webpack = require('webpack');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  entry: './index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'index.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ['@babel/preset-env', '@babel/preset-react']
+          }
+        }
+      },
+      {
+        test:/\.scss$/,
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader,
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              modules: true,
+              importLoaders: true,
+              localIdentName: '[folder]_[local]_[hash:base64:5]',
+              sourceMap: true,
+            },
+          },
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: true,
+            }
+          }
+        ],
+      },
+      {
+        test: /\.css$/,
+        use: [
+          "style-loader",
+          "css-loader"
+        ]
+      }
+    ]
+  },
+  resolve: {
+    extensions: ['.js'],
+    alias: {
+      'components': path.join(__dirname, 'components'),
+      'containers': path.join(__dirname, 'containers'),
+      'sass': path.join(__dirname, 'sass'),
+    },
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './index.html',
+      filename: 'index.html',
+    }),
+    new MiniCssExtractPlugin({
+      filename:'index.css'
+    }),
+    new webpack.DefinePlugin({
+      'process.env': {}
+    }),
+    // new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Is this commented code needed?

Reply:
not yet, but will do if we have bundles (ie multiple pages), so I left it there so I can uncomment when needed, i can remove though?

Reply:
no need to remove, just wanted to know why it was left